src/lib/flow/include/sol-flow-metatype.h

```diff
@@ -95,12 +95,12 @@ struct sol_flow_metatype_context {
      * @param ctx The metatype context.
      * @param name The file path.
      * @param buf Where the file contents should be stored.
-     * @param size The size of the buffer.
```

Comment:
You mean in the case that the _read_file_ would append data to the buffer? As the patch currently is, it is the read_file responsibility to init the sol_buffer